test rgw obc quota failure

Message: AssertionError: New quota {"spec": {"additionalConfig":{"maxObjects": "4"}}} didn't get applied!!
assert False
Type: None

E           ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n openshift-storage rsh session-awscli-relay-pod-a951593ab9c54d3 sh -c "AWS_CA_BUNDLE=/cert/service-ca.crt AWS_ACCESS_KEY_ID=***** AWS_SECRET_ACCESS_KEY=***** AWS_DEFAULT_REGION=us-east-1 aws s3 --endpoint=***** cp test_obc_quota[1-RGW-OC-quota0]/result/new-object-0 s3://rgw-oc-bucket-2606f21d5a9f4b42b5714099db".
E           Error is upload failed: test_obc_quota[1-RGW-OC-quota0]/result/new-object-0 to s3://rgw-oc-bucket-2606f21d5a9f4b42b5714099db/new-object-0 An error occurred (QuotaExceeded) when calling the PutObject operation: Unknown

--- a/ocs_ci/ocs/resources/objectbucket.py
+++ b/ocs_ci/ocs/resources/objectbucket.py
@@ -489,7 +489,8 @@ class RGWOCBucket(OCBucket):
             self.name = create_unique_resource_name("oc", "obc")
         obc_data["metadata"]["name"] = self.name
         obc_data["spec"]["bucketName"] = self.name
-        obc_data["spec"]["additionalConfig"] = self.quota
+        if self.quota:
+            obc_data["spec"]["additionalConfig"] = self.quota
         if storagecluster_independent_check():
             obc_data["spec"][
                 "storageClassName"

--- a/tests/manage/rgw/test_obc_quota.py
+++ b/tests/manage/rgw/test_obc_quota.py
@@ -7,12 +7,17 @@ from ocs_ci.ocs.bucket_utils import (
     copy_random_individual_objects,
 )
 from ocs_ci.ocs.exceptions import CommandFailed
-from ocs_ci.framework.pytest_customization.marks import tier1, bugzilla
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    bugzilla,
+    skipif_ocs_version,
+)
 
 logger = logging.getLogger(__name__)
 
 
 @bugzilla("1940823")
+@skipif_ocs_version("<4.10")
 class TestOBCQuota:
     """
     Test OBC Quota feature

